src/test/resources/p8private.pem

```diff
@@ -0,0 +1,8 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Can you add a README that explains how these files were generated?

Reply:
It's already there under the `Generating keys` section.

Reply:
Got it, thanks!